ts/Data/DataOnDemand.ts

```diff
@@ -117,7 +117,7 @@ class DataOnDemand {
             return this.loadStore(sourceOptions);
         }
 
-        throw new Error('Source not found');
+        throw new Error(`Source not found. (${name})`);
```

Comment:
How about?
```
`Source ${name} not found.
```

Reply:
This would make it harder to have a string comparison agains a specific error message. That's why related values are put at the end.